plugin-script-windows/src/main/java/io/kestra/plugin/scripts/windows/WindowsCmdTask.java

```diff
@@ -0,0 +1,158 @@
+package io.kestra.plugin.scripts.windows;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.slf4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Task to execute Windows commands using cmd.exe.
+ * <p>
+ * This task allows users to execute Windows commands by providing a command string
+ * and an optional working directory.
+ */
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@Schema(
+    title = "Execute Windows commands using cmd.exe",
+    description = "This task allows you to execute Windows commands using cmd.exe."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "List files in a directory",
+            code = {
+                "command: \"dir\""
+            }
+        ),
+        @Example(
+            title = "Run an executable file",
+            code = {
+                "command: \"C:\\\\path\\\\to\\\\program.exe\""
+            }
+        )
+    }
+)
+public class WindowsCmdTask implements RunnableTask<WindowsCmdTask.Output> {
+    /**
+     * Default constructor for WindowsCmdTask.
+     * Required for deserialization and object creation.
+     */
+    public WindowsCmdTask() {
+        // Default constructor
+    }
+
+    /**
+     * The Windows command to execute using cmd.exe.
+     */
+    @Schema(
+        title = "Command to execute",
+        description = "The Windows command to execute using cmd.exe."
+    )
+    @PluginProperty(dynamic = true)
+    private String command;
```

Comment:
The annotation `@PluginProperty` is the old way to do. Can you please for all properties do like this?

```suggestion
    private Property<String> command;
```